dotcom-rendering/src/web/components/CommercialMetrics.tsx

```diff
@@ -15,43 +15,31 @@ export const CommercialMetrics: React.FC<{
 	const ABTestAPI = useAB();
 	const visibilityState = useDocumentVisibilityState();
 
-	const [sentCommercialMetrics, setSentCommercialMetrics] = useState<boolean>(
-		false,
-	);
-
-	const adBlockerInUse = useAdBlockInUse()
-
-	// TODO replace useEffect with useOnce, simplifying the below logic
-	useEffect(() => {
-		if (visibilityState !== 'hidden') return;
-		if (sentCommercialMetrics) return;
+	const adBlockerInUse = useAdBlockInUse();
+	const isHidden = visibilityState === 'hidden';
```

Comment:
Maybe something like this could make it clearer?

```suggestion
	const isHidden = visibilityState === 'hidden' ?? undefined;
```